v62000/u_inref/gtm8086.csh

```diff
@@ -145,4 +146,13 @@ while ($jobnum < $gtm_test_jobcnt)
 	$grep -vE 'JNLEXTEND|JNLCLOSE|NOTALLDBRNDWN|GVRUNDOWN' impjob_imptp1.xmje${jobnum}x > impjob_imptp1.mje${jobnum}
 end
 
+if ($?test_replic) then
+	if ($jnlswitchretry) then
+		# getoper will capture syslog messages generated since starting the jnlswitchretry error
+		$gtm_tst/com/getoper.csh "$jnlswitchretry_time"
+		# Confirm there is no PREVJNLLINKCUT error in the output
```

Comment:
Add a comment here too about "tests GTM-8699 in V63004" and another line mentioning the exact line item being tested i.e. "Previously the [PREVJNLLINKCUT error] message could be reported in rare cases where a journal switch was deferred, then later switched without cutting links"

Reply:
The appropriate comments have been added to lines 150 and 151